README.md

```diff
@@ -215,8 +216,42 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
   * **Frequency**: Once a week
   * **Runtime**: 30 - 40 mins, regularly ~35 mins
 
+## Data Management
+
+* [Big Data Beard](https://bigdatabeard.com/subscribe-to-podcast/)
+  * **Description**: Sharing crumbs of knowledge and experience collected from working in the big data ecosystem.
+  * **Frequency**: Once every week
+  * **Host**: Cory Minto, et al.
+  * **Runtime**: 30 - 60 min, regularly ~45 mins
+
+* [Data Engineering Podcast](https://www.dataengineeringpodcast.com)
+  * **Description**: A show that digs deep into databases, big data pipelines, data governance, data collection, ETL, building effective data teams, and all of the other challenges faced by technology professionals working on data management.
+  * **Frequency**: Once every week
+  * **Host**: Tobias Macey
+  * **Runtime**: 30-60 mins, regularly ~45 mins
+
+* [Drill To Detail](https://www.drilltodetail.com/)
+  * **Description**: A podcast and accompanying blog on the business and strategy of analytics, big data and distributed processing in the cloud
+  * **Frequency**: Varies
+  * **Host**: Mark Rittman
+  * **Runtime**: 30-60 mins, regularly ~45 mins
+
+* [Roaring Elephant](https://roaringelephant.org/)
+  * **Description**: A weekly community podcast about Apache Hadoop and the surrounding ecosystem for anyone working with or investigating Big Data and Advanced Analytics
+  * **Frequency**: Once every week
+  * **Host**: Jhon Masschelein and Dave Russell
+  * **Runtime**: 15 - 60 mins, regularly ~20 mins
+
+
 ## Data Science/Machine Learning
 
+* [The Architecht Show](http://architechtshow.com/)
```

Comment:
Not alphabetical order

Reply:
Sorry, I was going by the convention of omitting `The` as a meaningful word for alphebetizing. I'll update the ordering now.